server/index.ts

```diff
@@ -1,7 +1,7 @@
 // Import the express in typescript file
 import express from "express";
 import http from "http";
-import { Server } from "socket.io";
+import { Server, Socket } from "socket.io";
```

Comment:
Importing `Socket` can be removed once the `Trade` class is moved into its own file.
You'll also want to make its own socket event format in `common/messages` so that it doesn't have access to *every* single event.